src/modules/discovery/pages/landing/views/defaultView/components/TrendingRewardCard.tsx

```diff
@@ -72,13 +76,21 @@ export const TrendingRewardCard = ({ reward, ...rest }: TrendingRewardCardProps)
                   $
                 </Body>{' '}
                 {reward.cost < 100 ? (reward.cost / 100).toFixed(2) : commaFormatted(Math.round(reward.cost / 100))}
+                <Body as="span" size="sm" light>
+                  {' '}
+                  ({formatSatsAmount(reward.cost)})
+                </Body>
               </>
             ) : (
               <>
                 {commaFormatted(reward.cost)}{' '}
                 <Body as="span" size="sm" light>
                   sats
                 </Body>
+                <Body as="span" size="sm" light>
+                  {' '}
+                  (${formatUsdAmount(reward.cost)})
```

Comment:
I think this would case an extra $ to be rendered. Not sure, but please confirm.

Reply:
It does not